package.json

```
@@ -5,13 +5,15 @@
   "scripts": {
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
-    "test": "truffle test",
+    "test": "npx hardhat test --show-stack-traces",
```

`npx` isn't necessary here because running `npm test` already fixes the `PATH`.

```suggestion
    "test": "hardhat test --show-stack-traces",
```